cloud/azure/README.md

```diff
@@ -0,0 +1,68 @@
+# SignalFx Azure Integration
+
+
+## Usage
+
+```hcl
+terraform {
+  required_version = ">= 0.12"
+  required_providers {
+    azurerm  = "~> 1.44"
+    azuread  = "~> 0.7"
+    signalfx = "~> 4.16"
+  }
+}
```

Comment:
Probably superfluous as that's already enforced in the module: https://github.com/claranet/terraform-signalfx-integrations/pull/1/files#diff-89fbb1cc33f1f1848e81d38634f0a0dcR2

In a root configuration, you'd more likely pin versions.

Reply:
I will recreate every readme with a script as it is done on datadog ;)